### PR TITLE
Add newlines in pull request comment table

### PR DESCRIPTION
**Pull request by omerzi**
- [x] All [tests](https://github.com/jfrog/frogbot#tests) passed. If this feature is not already covered by the tests, I added new tests.
- [x] This pull request is on the dev branch.
- [x] I used gofmt for formatting the code before submitting the pull request.
---
* For 'Direct Dependencies', 'Direct Dependencies Versions', and 'Fixed Versions' columns, the arguments will be separated by newlines.

### CODE REVIEW

File: commands/utils/simplifiedoutput.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+import (
+	"fmt"
+	"github.com/jfrog/jfrog-cli-core/v2/xray/formats"
+	"strings"
+)
+
+type SimplifiedOutput struct{}
+
+func (smo *SimplifiedOutput) TableRow(vulnerability formats.VulnerabilityOrViolationRow) string {
```
Comment (eyalbe4) [inline commands/utils/simplifiedoutput.go:11]: Let's add a unit test for this function.

File: commands/utils/standardoutput.go [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+import (
+	"fmt"
+	"github.com/jfrog/jfrog-cli-core/v2/xray/formats"
+	"strings"
+)
+
+type StandardOutput struct{}
+
+func (so *StandardOutput) TableRow(vulnerability formats.VulnerabilityOrViolationRow) string {
```
Comment (eyalbe4) [inline commands/utils/standardoutput.go:11]: Let's add a unit test for this function.

File: commands/utils/utils.go [hunk 3]
```diff
@@ -37,6 +38,14 @@ func (e *ErrMissingConfig) Error() string {
 	return fmt.Sprintf("config file is missing: %s", e.missingReason)
 }
 
+type OutputWriter interface {
```
Comment (eyalbe4) [inline commands/utils/utils.go:41]: Let's add a description for each of the functions in this interface.

File: commands/utils/utils.go [hunk 4]
```diff
@@ -137,6 +146,9 @@ func GetRelativeWd(fullPathWd, baseWd string) string {
 	return strings.TrimPrefix(fullPathWd, baseWd+string(os.PathSeparator))
 }
 
-func isSimplifiedOutput(provider vcsutils.VcsProvider) bool {
-	return provider == vcsutils.BitbucketServer
+func SetOutputWriter(provider vcsutils.VcsProvider) OutputWriter {
```
Comment (eyalbe4) [inline commands/utils/utils.go:149]: SetOutputWriter --> GetCompatibleOutputWriter

**Review by eyalbe4 — CHANGES_REQUESTED** [review]

File: commands/utils/utils.go (hunk 3 above)
Comment (omerzi) [reply commands/utils/utils.go:41]: done

File: commands/utils/utils.go (hunk 4 above)
Comment (omerzi) [reply commands/utils/utils.go:149]: done

**Review by eyalbe4 — APPROVED** [review]

**Pull request merged**